lib/neo4j/active_node/query/query_proxy_methods.rb

```diff
@@ -255,11 +255,12 @@ def where_clause_params
         end
 
         def first_and_last(func, target)
+          ord_prop = (func == LAST ? {order_property => :DESC} : order_property)
```

Comment:
Since `ord_prop` is only used inside the `else`, it should be able to be defined there.  Probably not a big deal, but it should reduce computation in some cases.  I'll make this change after merging.